IDEAS/Fluid/Taps/Examples/package.order

```diff
@@ -1 +1,3 @@
+Tap
+TapProfile
```

Comment:
We don't have an example showing how to calculate thermal discomfort (e.g. in [Kh]). I would add this to the output of at least one Example.